server/controllers/BookController.js

```diff
@@ -42,6 +42,33 @@ class BookController {
       return responseMessage(response, 500, { message: error.message });
     }
   }
+
+  /**
+* pagination
+* @name pagination
+* @param {Object} request
+* @param {Object} response
+* @returns {JSON} object
+*/
+  static async pagination(request, response) {
+    const { page, limit } = request.query;
+    let pageLimit = limit;
+    let offset = 0;
+    pageLimit = +pageLimit ? pageLimit : pageLimit = 10;
+    try {
+      const countResults = await Book.findAndCountAll();
+      const { count } = countResults;
+      const pages = Math.ceil(countResults.count / pageLimit);
```

Comment:
You already destructed count from countResults

Reply:
okay